integration-test-common/src/test/java/org/apache/gravitino/integration/test/container/ContainerSuite.java

```diff
@@ -71,6 +71,16 @@ public class ContainerSuite implements Closeable {
   private static volatile Map<PGImageName, PostgreSQLContainer> pgContainerMap =
       new EnumMap<>(PGImageName.class);
 
+  private static volatile GravitinoLocalStackContainer gravitinoLocalStackContainer;
+
+  /**
+   * We can share the same Hive container as Hive container with S3 contains the following
+   * differences: 1. Configuration of S3 and corresponding environment variables 2. The Hive
+   * container with S3 is Hive3 container and the Hive container is Hive2 container. There is
+   * something wrong with the hive2 container to access the S3.
+   */
+  private static volatile HiveContainer hiveContainerWithS3;
```

Comment:
We cannot share the container even if we uniform the hive version to hive3 for the following reasons.
1. We need to ensure the s3 environment is ready before starting the Hive container so we can share it.
2. for the reason in #(1, we need to start localstack(S3 simulator) first, however we can't make sure `CatalogHiveS3IT` was the first one to execute.